test_conformance/basic/test_explicit_s2v.cpp

```diff
@@ -13,138 +13,122 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+#include <cmath>
+using std::isnan;
+#define isnan isnan
```

Comment:
We can remove the define completely now.

```suggestion
using std::isnan;
```